.gitignore

```diff
@@ -0,0 +1,110 @@
+# Build Folders (you can keep bin if you'd like, to store dlls and pdbs)
```

Comment:
Always have a .gitignore. This lets you not upload unnecessary files such as Debug info in C# projects, or node_modules in a nodejs project. Things that are auto-generated or packages that can be fetched don't need to clog up the repo.